caffe2/contrib/tensorrt/tensorrt_tranformer.cc

```diff
@@ -40,6 +40,21 @@ std::unordered_map<std::string, TensorShape> InferShapes(
   return shape_hints;
 }
 
+void DumpModel(const ::ONNX_NAMESPACE::ModelProto& model) {
+  std::ofstream ff("trt.onnx");
+  for (const auto& t : model.graph().initializer()) {
+    ff << "tensor: " << t.name() << std::endl;
+    ff << "  dims: ";
+    for (auto i : t.dims()) {
+      ff << i << " ";
+    }
+    ff << std::endl;
+    for (auto i : t.float_data()) {
+      ff << "    " << i << std::endl;
```

Comment:
This is a lot of weights. Maybe print just first and last values in `float_data()`?

Reply:
First and last 10 might be more informative.